conf/quest.config

```diff
@@ -35,9 +38,15 @@ process {
 params {
             
     // genome
-    genome = "WS245"
-    reference = "/projects/b1059/data/genomes/c_elegans/${genome}/${genome}.fa.gz"
+    genome = "WS276"
```

Comment:
You can probably download WS276 and add it to the repo as it was before

Reply:
Will come back to this once I get around.